core/tests/protractor/collections.js

```diff
@@ -19,14 +19,17 @@
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var AdminPage = require('../protractor_utils/AdminPage.js');
-var collectionEditor = require('../protractor_utils/collectionEditor.js');
+var CollectionEditorPage = 
```

Comment:
Is this supposed to be here? I thought you were introducing this page object in: #4058